src/nodes.coffee

```diff
@@ -232,18 +232,20 @@ exports.Base = class Base
   cacheToCodeFragments: (cacheValues) ->
     [fragmentsToText(cacheValues[0]), fragmentsToText(cacheValues[1])]
 
-  # Construct a node that returns the current node's result.
+  # Construct a node that returns the current node’s result.
   # Note that this is overridden for smarter behavior for
-  # many statement nodes (e.g. If, For)...
-  makeReturn: ({accumulator, mark} = {}) ->
+  # many statement nodes (e.g. `If`, `For`).
+  makeReturn: (results, mark) ->
```

Comment:
I know it’s an older style, but the other methods in this class all take comma-separated arguments. I feel like we should be consistent.